### DISCUSSION TITLE
Is servings supported?

### DISCUSSION THREAD

**Discussion by selfire1** [cooklang/spec#70] [Q&A] [answered]
Hi! Just transferred a bunch of recipes to cooklang and am really excited about it!

In the docs it talks about servings in the "Roadmap" section. So am I correct in assuming it is not supported yet?

**Comment by dubadub** [comment, marked as answer]
Hi @selfire1, yes, that's correct. It's not yet supported, but one of the first things planned for [CLI](https://github.com/cooklang/CookCLI/milestones?direction=asc&sort=title&state=open). For the [iOS app](https://github.com/cooklang/cooklang-app-ios/milestones?direction=asc&sort=title&state=open) it's a bit further down the road. It's hard to give exact timeline when it's going to be released.

**Reply by selfire1** [reply]
Alright, good to know. Thank you!

**Comment by ekle** [comment]
Any news on this?

`./cook shopping-list Olivier\ Salad.cook*5` doesn't seem to work.

**Reply by dubadub** [reply]
fixed, you can use `./cook shopping-list Olivier\ Salad.cook:5` to set the scaling

**Reply by ekle** [reply]
That does seem to scale the recipe by a factor.

The recipe has `servings ` specified. How do i scale it to a different number of persons?
